abe/abe_internal_test.go

```diff
@@ -0,0 +1,92 @@
+package abe
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/fentec-project/gofe/data"
+	"github.com/fentec-project/gofe/sample"
+	"github.com/stretchr/testify/assert"
+)
+
+
+func TestBooleanToMsp(t *testing.T) {
```

Comment:
This function can be in package `abe_test` since `BooleanToMSP` is exported.

Reply:
Testing BooleanToMSP uses Gaussian elimination which is not public.

Reply:
Ah, sorry, I completely missed that.